src/ex1/index.html

```diff
@@ -1,14 +1,39 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <title>Exercise 1</title>
+  <head>
+    <title>Cool Todo List</title>
     <link rel="stylesheet" href="style.css" />
-</head>
-<body>
-
-<h1>Todo list</h1>
-
-
-</body>
-<script src="script.js"></script>
+  </head>
+  <body>
+    <fieldset class="container__field">
+      <div class="container">
+        <h1>Todo App</h1>
+        <form action="">
+          <div class="antd">
+            <input
+              class="ant-input"
+              id="input"
+              type="text"
+              name="todo"
+              placeholder="Add your new todo"
+            />
+            <input id="addButton" class="addbtn" type="button" value="+" />
```

Comment:
its better to use <button> for buttons, this way you'll have more control on the button, you will be able to put icons/images or whatever you'll want later on
also IMO its more readable and clear that way